### PR TITLE
3 small, but important updates

### PR DESCRIPTION
**Pull request by evaldsurtans**
1. Fixed doom_py integration - will work with and without it without need to uncomment code
2. Improved low dimensional representation for Snake and WaterWorld. Without new improvements learning from this representations would be unnecessarily difficult, because it was incomplete.

### CODE REVIEW

File: ple/games/snake.py [hunk 1]
```diff
@@ -1,405 +1,407 @@
-import pygame
```
Comment (ntasfi) [inline ple/games/snake.py]: What is the actual change here? Can you fix it so the _entire_ file is not being replaced

File: ple/ple.py [hunk 2]
```diff
@@ -1,416 +1,420 @@
-import numpy as np
-from PIL import Image  # pillow
```
Comment (ntasfi) [inline ple/ple.py]: Same comment as `snake.py`. Can you adjust this so only the relevant changes show up?

**Comment by evaldsurtans** [comment]
git client messed up commit, I will clean it up and submit for merge again

**Comment by ntasfi** [comment]
Closing as the changes were merged in another PR.

**Pull request closed without merging**